code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1055,6 +1055,10 @@
 	color = "#A5F0EE" // rgb: 165, 240, 238
 	taste_description = "sourness"
 	reagent_weight = 0.6 //so it sprays further
+	
+/datum/reagent/space_cleaner/on_mob_life(mob/living/carbon/M)
+	M.adjustToxLoss(5)
```

Comment:
```suggestion
	M.adjustToxLoss(0.25)
```

Considering this goes undetected from 'smart-purgers' and is made with ammonia and water, probably best to not have it crit in roughly 20 iterations.

Reply:
I am going to be adjusting it to completely hose your lungs and throat once I can figure out how organ damage works,

Reply:
lol

Reply:
I'm starting to suspect that this change might not actually be a good faith attempt to improve the game